packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.5
```

Comment:
nit: This should be `0.5.4+3`. Pub changes it's semver scheme for when the plugin is above or below 1.0.0. When it's above 1.0.0 it's `major.minor.patch` but when it's below it's `0.major.minor+patch`.

Reply:
Thanks for the catch, sorry about that!